definitions/npm/algoliasearch_v3.x.x/flow_v0.71.x-/algoliasearch_v3.x.x.js

```diff
@@ -64,8 +64,8 @@ declare type $algoliasearch$ClientOptions = {|
   protocol?: "http:" | "https:",
   httpClient?: HTTPClient,
   hosts?: {|
-    read?: string,
-    write?: string
+    read?: string | Array<string>,
+    write?: string | Array<string>
```

Comment:
Looks like it should be only `Array<string>`.    
Could you add/update tests in consequence?

Reply:
Algolia accepts borh string and array of strings. I will try to add tests, yet i have never tested flow typings before.

Reply:
All the docs, unit test, even the TypeScript definitions allows only `Array<string>`. 
Also [sources](https://github.com/algolia/algoliasearch-client-javascript/blob/ba1f66222acb381372850da1d7caa39fd3c67580/src/AlgoliaSearchCore.js#L60-L63).